Src/Database/Services/system/db_config.js

```diff
@@ -0,0 +1,11 @@
+export const HOST = "localhost";
```

Comment:
This file is not a service.
you need to move it to the database level.

Reply:
done it is in system folder